gnosis/spec-tmpl.k

```diff
@@ -106,16 +106,11 @@
     requires #rangeAddress(#MASTER_COPY_ID)
      andBool #rangeAddress(#PROXY_ID)
      andBool #rangeAddress(#ORIGIN_ID)
-     andBool #rangeUInt(256, MASTER_BAL)
-     andBool #rangeUInt(256, PROXY_BAL)
-     andBool #rangeUInt(256, ORIGIN_BAL)
      andBool #rangeUInt(256, NOW)
      andBool #rangeUInt(128, BLOCK_NUM) // Solidity
-     //
-     andBool #MASTER_COPY_ID =/=Int #PROXY_ID
-     andBool #MASTER_COPY_ID =/=Int #ORIGIN_ID
-     andBool #PROXY_ID       =/=Int #ORIGIN_ID
```

Comment:
why did you remove these?

Reply:
These constraints are mainly used for transferFunds. I think we'd better move these to each spec that involves transferFunds instead of putting it in the template.